lib/view/channel-item.js

```diff
@@ -86,7 +86,7 @@ class ChannelItem {
       attributes.font = UNREAD_FONT
       attributes.color = SELECTED_COLOR
     }
-    const textRect = Object.assign(bounds, {x: PADDING, y: 0})
```

Comment:
You removed the only reference to `PADDING` but left the assignment, btw.

Reply:
We should change the value of `PADDING` instead of changing the code directly, in future we will have theme support.

Also in official app the "Channels" title is aligned with channel names, we should do the same in Wey.